indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -141,9 +141,25 @@ static RealtimeTuningConfig convertTuningConfig(ShardSpec spec, IndexTuningConfi
 
   private final ObjectMapper jsonMapper;
 
+  private String taskId;
```

Comment:
Crap - that got left over from something else I was doing. Must have wound up in there with the rebasing and force pushing. Sorry, will remove.